src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -299,7 +299,23 @@ private boolean isNotificationEventValidToSendToJointParty() {
     private boolean isNotificationEventValidToSendToOtherPartySubscription(Subscription subscription, boolean isSendNewOtherPartyNotification) {
         return isValidSubscriptionOrIsMandatoryLetter(subscription, responseWrapper.getNotificationEventType())
                 && (DWP_UPLOAD_RESPONSE_NOTIFICATION.equals(getNotificationType())
-                || (isSendNewOtherPartyNotification && UPDATE_OTHER_PARTY_DATA.equals(getNotificationType())));
+                || ADJOURNED_NOTIFICATION.equals(getNotificationType())
```

Comment:
Do we not need a unit test to cover these new notifications that are sent?

Reply:
for UpdateOtherParty, they are in Rasika's branch that I am merging into. For the others. I added a few.